merlin-framework-processor/build.gradle

```diff
@@ -10,6 +10,7 @@ java {
 }
 
 dependencies {
+  implementation project(':merlin-sdk')
```

Comment:
This dependency seems to have been added in a commit called "Remove [...] dependencies" 🤔

Reply:
Somewhat frustratingly, it's not actually unnecessary -- the processor needs the merlin-sdk classes, and as best as I can tell, the compile-time classpath (the `compileOnly` dependency for models on `merlin-sdk`) is separate from the runtime classpath of the processor actively executing in the compiler's JVM. That is, the reflective APIs at processing time consider e.g. the `Duration` referenced by the model to be distinct from the `Duration` referenced by the processor.

I'll tweak the commit message to be more clear about what the dependency reshuffling is doing, though.